vignettes/epiprocess.Rmd

```diff
@@ -198,3 +198,89 @@ ggplot(x, aes(x = time_value, y = cases)) +
   scale_x_date(minor_breaks = "month", date_labels = "%b %y") +
   labs(x = "Date", y = "Confirmed cases of Ebola in Sierra Leone") 
 ```
+
+## Examples on Additional Keys in epi_df
```

Comment:
-> "## Using additional key columns in `epi_df`"?

And maybe move this to come immediately after the metadata section, since it's also metadata?